packages/venia-styleguide/src/pages/typography/sections/Formatting/index.md

```diff
@@ -0,0 +1,14 @@
+import Columns from "../../../../components/Columns"
+import Emphasis from "./Emphasis"
+import Strong from "./Strong"
+
+Semantic text formatting adds importance or emphasis to words or phrases.
+
+***
+
+<Columns reverse={true}>
+  <Strong />
+</Columns>
+<Columns reverse={true}>
+  <Emphasis />
+</Columns>
```

Comment:
This looks weird -- should the text be contained within the grey-box background? 

[![Image from Gyazo](https://i.gyazo.com/5fa4bef7d3d90e5853e72e9c5df1198f.png)](https://gyazo.com/5fa4bef7d3d90e5853e72e9c5df1198f)